lib/active_interactor/organizer/interactor_interface.rb

```diff
@@ -78,8 +83,31 @@ def execute_inplace_callback(target, callback)
         resolve_option(target, callbacks[callback])
       end
 
+      def execute_after_perform_callbacks(context)
+        return unless after_perform_callbacks.present?
+
+        interactor = interactor_class.new(context)
+        env = ActiveSupport::Callbacks::Filters::Environment.new(interactor, false, nil)
+        after_perform_callbacks.compile.invoke_after(env)
+        interactor.send(:context)
+      end
+
       private
 
+      def init_after_perform_callbacks
+        after_callbacks_deferred = @interactor_class.present? 
+                                && @interactor_class.after_callbacks_deferred_when_organized
+        @after_perform_callbacks = after_callbacks_deferred ? @interactor_class._perform_callbacks : nil
+      end
+      
+      def skip_after_perform_callbacks
+        return unless after_perform_callbacks.present?
+
+        after_perform_callbacks.each do |callback|
+          interactor_class.skip_callback(:perform, :after, callback.filter, raise: false)
```

Comment:
This was throwing me off a tiny bit. But I think I get it. Maybe we can rename the method to follow something closer to `after_callbacks_deferred_when_organized` as I wasn't quite seeing they were related right up front?

Reply:
Maybe I could rename `after_perform_callbacks` to `deferred_after_perform_callbacks`...would that help?  I'll make that change, let me know if it clears it up.

Reply:
I made that change...let me know if that clears it up.

Reply:
Sounds good, thanks!